AUTHORS.rst

```diff
@@ -13,3 +13,6 @@ In alphabetical order:
 - Markus Unterwaditzer
 - Michael Adler
 - Thomas Weißschuh
+
+Additionally `FastMail sponsored a paid account for testing
```

Comment:
>  Something of value, such as a ticket for admission or a book, given free of charge
[source](http://www.thefreedictionary.com/comped).

I didn't either, but that's the wording used for the offer. :)